src/Cosmonaut/CosmosStoreSettings.cs

```diff
@@ -19,14 +23,16 @@ public class CosmosStoreSettings
 
         public IndexingPolicy IndexingPolicy { get; set; } = CosmosConstants.DefaultIndexingPolicy;
 
-        public int DefaultCollectionThroughput { get; set; } =  CosmosConstants.MinimumCosmosThroughput;
+        public int DefaultCollectionThroughput { get; set; } = CosmosConstants.MinimumCosmosThroughput;
 
         public JsonSerializerSettings JsonSerializerSettings { get; set; }
 
         public bool InfiniteRetries { get; set; } = true;
 
         public string CollectionPrefix { get; set; } = string.Empty;
 
+        public List<IQueryInterceptor> Interceptors { get; }
```

Comment:
Can you change this to an `IReadOnlyList`? I don't think this should be mutable as it can cause unintended chaos.

Reply:
Superseded, interceptors to be moved to CosmosStore and renamed "QueryFilters"